runtime/lua/vim/diagnostic.lua

```diff
@@ -631,7 +631,7 @@ local function diagnostic_move_pos(opts, pos)
   local win_id = opts.win_id or vim.api.nvim_get_current_win()
 
   if not pos then
-    vim.api.nvim_echo({"No more valid diagnostics to move to", "WarningMsg"})
+    vim.api.nvim_echo({"No more valid diagnostics to move to", "WarningMsg"}, true, {})
```

Comment:
try this:

 vim.api.nvim_echo({{"No more valid diagnostics to move to", "WarningMsg"}}, true, {})
Write

Reply:
Ah, I was missing the outer braces -- thanks! (I've not finished my morning coffee yet...)